user_docs/en/userGuide.t2t

```diff
@@ -2089,6 +2089,21 @@ The tone will increase in pitch every space, and for a tab, it will increase in
 - Both Speech and Tones: This option reads indentation using both of the above methods.
 -
 
++++ Document Navigation +++[DocumentNavigation]
+This category allows you to adjust various aspects of document navigation.
+
+==== Paragraph Style ====[DocumentFormattingParagraphStyle]

```

Comment:
We try to avoid including section anchors in these names, as settings may move sections

```suggestion
==== Paragraph Style ====[ParagraphStyle]
```